core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -298,22 +275,17 @@ var ExplorationPlayerPage = function() {
 
   this.expectExplorationRatingOnInformationCardToEqual = async function(
       ratingValue) {
-    await explorationInfoIcon.click();
+    await action.click('Exploration Info Icon', explorationInfoIcon);
     var value = await infoCardRating.getText();
     expect(value).toBe(ratingValue);
   };
 
   this.rateExploration = async function(ratingValue) {
     var elements = ratingStars;
-    await waitFor.elementToBeClickable(
-      await elements.get(ratingValue - 1),
-      'Rating Star takes too long to be clickable');
-    await (await elements.get(ratingValue - 1)).click();
+    await action.click('Submit Button', await elements.get(ratingValue - 1));
```

Comment:
```suggestion
    await action.click('Submit Button', ratingStars.get(ratingValue - 1));
```

You don't need an await for get() because it doesn't retun a promise (see [ref](https://www.protractortest.org/#/api?view=ElementArrayFinder.prototype.get)).

After you do this, you can get rid of "elements" variable above.